### ISSUE TITLE
padding=100

### ISSUE THREAD

**Issue by LynnHg** [wkentaro/pytorch-fcn#138]
would you mind tell me why is the first conv's padding =100?( self.conv1_1 = nn.Conv2d(3, 64, 3, padding=100))

**Comment by wkentaro** [comment]
Please refer to https://github.com/wkentaro/pytorch-fcn/issues/40

**Comment by LynnHg** [comment]
thanks！

**Comment by Niloy-Chakraborty** [comment]
Do you know how to add padding=100, in Keras?
